librz/core/p/core_java.c

```diff
@@ -1938,12 +1598,66 @@ static bool rz_cmd_java_handle_print_exceptions(RzCore *core, const char *input)
 	return true;
 }
 
+RZ_IPI RzCmdStatus rz_cmd_java_handler(RzCore *core, int argc, const char **argv) {
+	bool res = false;
+	for (int i = 0; JAVA_CMDS[i].name; i++) {
+		if (!strncmp(argv[1], JAVA_CMDS[i].name, JAVA_CMDS[i].name_len)) {
+			res = JAVA_CMDS[i].handler(core, argc - 2, argc > 2 ? &argv[2] : NULL);
```

Comment:
It would be awesome if the sub-handlers could directly return a RzCmdStatus, so for example you can return `RZ_CMD_STATUS_WRONG_ARGS` when the wrong number of arguments is passed to the sub-command.

Reply:
yes, i was thinking to do this. but then i just adapted the old ones to the current one. i will fix this tonight.